docs/install/command-line.md

```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
 
 Manually compiling IPFS is a fairly involved process that changes frequently. It can be handy if you'd like to build a specific branch or use the _bleeding-edge_ version of Go-IPFS. See the [`ipfs/go-ipfs` GitHub repository for details →](https://github.com/ipfs/go-ipfs)
 
+## Which node should you use with the command line
+
+The command line can detect and use any node that's running (unless it's configured to use an external binary file). Here's which node to use for the local daemon or a remote client:
+
+### Local daemon
+
+The local daemon process is automatically started with the ipfs daemon. It creates an `$IPFS_PATH/api` file with an [RPC API](./reference/http/api/#http-rpc-api-reference) address.
+
+### Remote client
+
+You can install the standalone CLI client independently and use it to talk to an IPFS Desktop node or a Brave node (either Release or Nightly). Use the [RPC API](./reference/http/api/#http-rpc-api-reference) to talk to the daemon.
+
+When an IPFS command is executed without parameters, the CLI client checks whether the `$IPFS_PATH/api` file exists and connects to the address listed there.
+
+- If an `$IPFS_PATH` is in the default location (for example, `~/.ipfs` on Linux), then it works automatically and the ipfs CLI client talks to the locally running ipfs daemon without any additional configuration.
+
+- If an `$IPFS_PATH` is not in the default location, pass `--api <rpc-api-addr>` or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.
```

Comment:
```suggestion
- If an `$IPFS_PATH` is not in the default location, use the `--api <rpc-api-addr>` command-line argument, or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.
```

> or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.

This part of the sentence is pretty long and hard to parse. Can you simplify things a little bit?